Podfile.lock

```diff
@@ -18,8 +18,8 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   Sourcery: 47b30d2494896127ff264ecb19efb3643521fcb9
-  SwiftyMocky: f3ac32a103b6fd945e08cb2d3a88a03185825737
+  SwiftyMocky: f327448da3a46db509fe09451e5cf01995d8f94d
 
 PODFILE CHECKSUM: 024f8b7ce6c9eed2b70ca54b9fcecda8c6cb6381
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.12.0
```

Comment:
Oleksii: why do we decrease the pods version?

Reply:
You are right, will revert this change.